windows/src/desktop/kmshell/xml/keyman_hotkeys.xsl

```diff
@@ -105,7 +105,7 @@
           <xsl:when test="Target='1'"><xsl:value-of select="$locale/string[@name='S_Hotkey_OpenKeyboardMenu']"/></xsl:when>
           <xsl:when test="Target='2'"><xsl:value-of select="$locale/string[@name='S_Hotkey_ShowOnScreenKeyboard']"/></xsl:when>
           <xsl:when test="Target='3'"><xsl:value-of select="$locale/string[@name='S_Hotkey_OpenConfiguration']"/></xsl:when>
-          <xsl:when test="Target='4'"><xsl:value-of select="$locale/string[@name='S_Hotkey_ShowKeyboardUsage']"/></xsl:when>
+          

```

Comment:
So the Targets 5-7 don't need to re-index?

Reply:
No, because those Target identifiers are published in the Keyman COM API, so they can't change.